IntroducingMethods/src/cst1201/IntroducingMethods.java

```diff
@@ -5,31 +5,47 @@
 /**
  * TODO: Description of your class.
  *
- * @author TODO: Your name here.
+ * @author TODO: razia and jean.
  */
 public class IntroducingMethods {
 
     public static void main(String[] args) {
 
-        // TODO: Fill in the body with your code
+        System.out.print(" Please enter a string: ");
+
+        Scanner keyboard = new Scanner(System.in);
+        getInputString(keyboard);
+
     }
 
-    // Given a Scanner, prompt the user for a String.  If the user enters an empty
-    // String, report an error message and ask for a non-empty String.  Return the
-    // String to the calling program.
     private static String getInputString(Scanner inScanner) {
-        // TODO: Fill in the body
 
-        // NOTE: Do not declare a Scanner in the body of this method.
+        String input = inScanner.nextLine();
+
+        while (input.length() == 0) {
+
+            System.out.println("ERROR - string must not be empty.");
+            System.out.println("Enter a string: ");
+
+            input = inScanner.nextLine();
+
+            getWordCount(input);
```

Comment:
Should not be called from here. This method **just** gets the input string, no less no more.